official/utils/arg_parsers/parsers.py

```diff
@@ -117,6 +118,18 @@ def __init__(self, add_help=False, data_dir=True, model_dir=True,
           help="If set, run across all available GPUs."
       )
 
+    if hooks:
+      self.add_argument(
+          "--hooks", "-hk", nargs="+", default=["LoggingTensorHook"],
+          help="[default: %(default)s] A list of strings to specify the names "
+               "of train hooks. "
+               "Example: --hooks LoggingTensorHook ExamplesPerSecondHook. "
```

Comment:
If the number of allowed hooks becomes large it may be worth it to tie this into hooks_helper through the choices arg of add_argument(). Probably overkill for now.